.github/workflows/ci.yml

```diff
@@ -107,6 +107,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Prefetch grammars
+        run: cargo run --all-features -- prefetch
+
```

Comment:
Should we keep this here? I only put it in to demonstrate the problem, but it's not necessary for the test suite. (Also, it prevents the test suite from fetching-and-building grammars; but it's the same code path, so that shouldn't matter.)